nuget.config

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="utf-8"?>
+<configuration>
+  <packageSources>
+    <add key="PowerShell-Module-Feed" value="https://powershell.myget.org/F/powershellmodule/api/v2" />
```

Comment:
Here we are using https://powershell.myget.org/F/powershellmodule/api/v2 and I saw somewhere we are using https://powershell.myget.org/gallery/powershell-core ?

Reply:
c44c680